playbook.yml

```diff
@@ -272,10 +341,17 @@
         enabled=yes
 
     - name: Service | Ensure Apache is restarted
+      when: server_webserver == "apache"
       service:
         name={{ apache_daemon }}
         state=restarted
 
+    - name: Service | Ensure Nginx is restarted
+      when: server_webserver == "nginx"
+      service:
+        name={{ nginx_daemon }}
+        state=restarted
+
```

Comment:
Another chance to use variables...  Let's change "apache_daemon" to "web_daemon"